### ISSUE TITLE
1.33.1: Linux systemctl stop nextdns hangs until timeout

### ISSUE THREAD

**Issue by timkgh** [nextdns/nextdns#524]
**Context**
 - CLI Version: 1.33.1
 - Platform: Ubuntu 20.04.2 amd64

```
Jun 19 15:44:47 penguin systemd[1]: Stopping NextDNS DNS53 to DoH proxy....
Jun 19 15:46:17 penguin systemd[1]: nextdns.service: State 'stop-sigterm' timed out. Killing.
Jun 19 15:46:17 penguin systemd[1]: nextdns.service: Killing process 2214621 (nextdns) with signal SIGKILL.
Jun 19 15:46:17 penguin systemd[1]: nextdns.service: Killing process 2214625 (nextdns) with signal SIGKILL.
Jun 19 15:46:17 penguin systemd[1]: nextdns.service: Main process exited, code=killed, status=9/KILL
Jun 19 15:46:17 penguin systemd[1]: nextdns.service: Failed with result 'timeout'.
Jun 19 15:46:17 penguin systemd[1]: Stopped NextDNS DNS53 to DoH proxy..
```

**Comment by timkgh** [comment]
Not only this, but even worse, 1.33.1 doesn't respond to any DNS requests, it just times out.

**Comment by rs** [comment]
Fixed in 1.33.2, sorry about that.
